composer/installed.json

```diff
@@ -2637,5 +2637,46 @@
             "cache",
             "caching"
         ]
+    },
+    {
+        "name": "deepdiver1975/tarstreamer",
+        "version": "dev-master",
+        "version_normalized": "9999999-dev",
+        "source": {
+            "type": "git",
+            "url": "https://github.com/VicDeo/TarStreamer.git",
+            "reference": "34aeb3ff15237bb152410bba5d9d25da0ac20568"
+        },
+        "dist": {
+            "type": "zip",
+            "url": "https://api.github.com/repos/VicDeo/TarStreamer/zipball/34aeb3ff15237bb152410bba5d9d25da0ac20568",
```

Comment:
once we know this solution works we should consider moving this to the owncloud orga